src/common/Section/style.js

```diff
@@ -1,24 +1,42 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 export const Container = styled.section`
   display: grid;
   align-items: center;
   justify-items: center;
   grid-template-columns: repeat(4, 324px);
   grid-gap: 24px;
+
   @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px) {
     grid-template-columns: repeat(2, 324px);
   }
+
   @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}px) {
     grid-template-columns: 1fr;
   }
+
+  ${(props) =>
+    props.people &&
+    css`
```

Comment:
Wouldn't it be better to write in one line?

Reply:
Maybe, but I have "Prettier" installed - it formats the code this way, so I didn't change it 🙂